_studio/mfx_lib/decode/CMakeLists.txt

```diff
@@ -20,118 +20,156 @@
 
 set(MFX_ORIG_LDFLAGS "${MFX_LDFLAGS}" )
 
-mfx_include_dirs()
-
-list( APPEND Decoders h264 h265 mpeg2 vp8 vp9 mjpeg vc1 )
-foreach( dir ${Decoders} )
-  include_directories( ${MSDK_LIB_ROOT}/decode/${dir}/include )
-  include_directories( ${MSDK_UMC_ROOT}/codec/${dir}_dec/include )
+list( APPEND vdirs
+  h264 h265 mjpeg mpeg2 vc1 vp8 vp9 av1
+  )
+list( APPEND cdirs
+  h265_dec h264_dec mpeg2_dec vp9_dec av1_dec vc1_dec jpeg_dec vc1_common color_space_converter jpeg_common
+  )
+mfx_include_dirs( )
+foreach( dir ${vdirs} )
+  include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/${dir}/include )
 endforeach()
-include_directories( ${MSDK_LIB_ROOT}/vpp/include )
-include_directories( ${MSDK_UMC_ROOT}/codec/jpeg_dec/include )
-include_directories( ${MSDK_UMC_ROOT}/codec/jpeg_enc/include )
-include_directories( ${MSDK_UMC_ROOT}/codec/jpeg_common/include )
-include_directories( ${MSDK_UMC_ROOT}/codec/vc1_common/include )
-include_directories( ${MSDK_UMC_ROOT}/codec/color_space_converter/include )
+include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/../vpp/include )
 
-set( sources.plus "" )
+foreach( dir ${cdirs} )
+  include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/../../shared/umc/codec/${dir}/include )
```

Comment:
put includes below definition of UMC_CODECS and reuse it for simplicity

Reply:
Done.